CRM/CiviAwards/Setup/CreateApplicantReviewActivityType.php

```diff
@@ -5,6 +5,8 @@
  */
 class CRM_CiviAwards_Setup_CreateApplicantReviewActivityType {
 
+  const APPLICANT_REVIEW = 'Applicant Review';
```

Comment:
Don't we need localisation tag for this?

Reply:
Will add for where this is used in labels.